README.md

```diff
@@ -47,14 +47,16 @@ Known to work with:
 
 TL;DR
 
-### Install from PyPi
+Doc ref: [pip install](https://pip.pypa.io/en/stable/cli/pip_install/)
 
+### Install from PyPi, along with required packages
 
-    python -m pip install pyodbc sqlalchemy-ingres
+
+    python -m pip install sqlalchemy-ingres[all]
```

Comment:
recommend double quoting this. I believe this runs cleanly under Windows but `[]` can sometimes be special characters in some shells so double quoting would work everywhere and be safer (possibly too protective ;-))

Reply:
Done. Thanks for the explanation on why the double quotes should be there.